server/mldb_server.cc

```diff
@@ -72,10 +72,11 @@ MldbServer::
 MldbServer(const std::string & serviceName,
            const std::string & etcdUri,
            const std::string & etcdPath,
-           bool enableAccessLog)
```

Comment:
Consider moving enableAccessLog and httpBaseUrl into the init method. Otherwise I think we should remove the default values.